README.md

```diff
@@ -56,9 +56,19 @@ Before installing `manim-community`, there are some additional dependencies that
 > win_amd64 corresponds to 64-bit machines, win32 corresponds to 32-bit machines
 
 3. Open up your command prompt by hitting the Win key and typing `cmd`
+
 4. Enter the directory where you install cairo by typing `cd C:\path\to\cairo` with the path being where you downloaded the `.whl` file
+
 5. Finally, run `pip3 install (cairo-file).whl`, where the file the appropriate name of the `.whl` file.
 
+6. Alternatively, Running the command below installs pycairo. This needs to be an elevated command prompt like Powershell.
```

Comment:
I don't understand why you would include as installation instructions that would point to your personal gist, when the purpose of this PR is to include the code from that gist inside the codebase. There is no need to point to your gist if the code will live inside this codebase. Please delete the mention of your gist and add installation instructions that would use the code inside `scripts/pycairoInstall.py`.

Finally, regardless of what happens to this particular PR, it is my personal opinion that the README should _ever_ point at someone's personal gist.

Reply:
I would change it.